.github/workflows/build/action.yaml

```diff
@@ -0,0 +1,27 @@
+name: 'openpilot build'
```

Comment:
I would call this action "compile-openpilot", build can be ambiguous (docker build, scons, etc.)